cmd/vic-machine/create/create.go

```diff
@@ -499,14 +499,19 @@ func (c *Create) processParams() error {
 		return err
 	}
 
-	if c.BridgeNetworkName == "" {
-		c.BridgeNetworkName = c.DisplayName
+	// prevent usage of special characters for certain user provided values
+	if err := common.CheckUnsupportedChars(c.DisplayName); err != nil {
```

Comment:
I think image store and volume store contains path name, and is not checked in vic-machine. It will be ideal to add check for them as well.
thanks